pandas/core/frame.py

```diff
@@ -1140,7 +1140,7 @@ def _repr_html_(self) -> str | None:
     def to_string(
         self,
         buf: None = ...,
-        columns: Axes | None = ...,
+        columns: IndexLabel | None = ...,
```

Comment:
I wasn't sure if the `| None` should be kept. Theoretically, `IndexLabel` is sufficient since `None` is `Hashable`. However, the `| None` makes it clearer that the value can be `None`.